<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoxelDialog</class>
 <widget class="QDialog" name="VoxelDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>242</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Voxel Aspect</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/ui/icons/vex_logo.png</normaloff>:/ui/icons/vex_logo.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QWidget" name="bottomWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <spacer name="leftHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pbApply">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="shortcut">
         <string>Enter</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pbCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="bottomVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <spacer name="topVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="spacingWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="gbSpacing">
        <property name="title">
         <string>Voxel spacing:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QLabel" name="lblMultSymbol1">
           <property name="text">
            <string>×</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="leSizeX">
           <property name="text">
            <string>1.0</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="leSizeY">
           <property name="text">
            <string>1.0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLineEdit" name="leSizeZ">
           <property name="text">
            <string>1.0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="lblMultSymbol2">
           <property name="text">
            <string>×</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leSizeX</tabstop>
  <tabstop>leSizeY</tabstop>
  <tabstop>leSizeZ</tabstop>
  <tabstop>pbApply</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>VoxelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbApply</sender>
   <signal>clicked()</signal>
   <receiver>VoxelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotApply()</slot>
  <slot>slotCancel()</slot>
 </slots>
</ui>
